domain/snyk/scanner.go

```diff
@@ -32,6 +32,11 @@ import (
 	"github.com/snyk/snyk-ls/internal/product"
 )
 
+var (
```

Comment:
What is this used for?

Reply:
These enforce that the class implements the `Scanner` and the `InlineValueProvider` interfaces. It's a pattern in go to ensure at compile time.